Sources/GoodExtensions/Array.swift

```diff
@@ -45,5 +45,14 @@ public extension Array where Element: Equatable {
             append(object)
         }
     }
-    
+
 }
+
+public extension Array {
+
+    var hasItems: Bool {
```

Comment:
Missing GRActive extension + deprecation alert as we did before (https://github.com/GoodRequest/GoodIOSExtensions/blob/04b7eb3f84bcc748d83623dd434a2be4391e1421/Sources/GoodExtensions/Array.swift)